test/unit/driver-specs.js

```diff
@@ -18,12 +19,36 @@ describe('driver commands', () => {
     proxySpy.reset();
   });
 
-  describe('status', () => {
-    it('should send status request to WDA', async () => {
-      await driver.getStatus();
-      proxySpy.calledOnce.should.be.true;
-      proxySpy.firstCall.args[0].should.eql('/status');
-      proxySpy.firstCall.args[1].should.eql('GET');
+  describe('status caching', () => {
```

Comment:
Yeah, Travis has been overloaded of late. But the unit tests and eslint ought to be quick, and catch much.